js/main.js

```diff
@@ -1 +1,38 @@
-/* Javascript goes here! */
+$(document).ready(function(){
+
+$('form').on('submit', function(event) {
+  event.preventDefault();
+
+    $.ajax({
+      method: 'GET',
+      url: 'http://api.openweathermap.org/data/2.5/weather',
+      //?q=${myCity}&&appid=373b383635979fcc983487687a41a48a`,
+      data: $("form").serialize(),
+      dataType: 'json',
+      success: onSuccess,
+      error: onError
+    })
+  });
+
+  function onSuccess(data) {
+    console.log("Success");
+    console.log(data);
+    var myCity = $('.results-city').text(data.name);
+
+    var iconId = (data.weather.id);
+    var temp = (data.main.temp);
+    var fahren = (9/5) * (temp - 273) + 32;
+    var humid = (data.main.humidity);
+    $('.temperature').text(Math.round(fahren) + " degrees Fahrenheit");
```

Comment:
Instead of doing all the math, you could have also used the API which requires another parameter to be sent as data in your ajax call 'units=imperial'. Since you are serializing your form to get the data automatically, you could have added another hidden input for this `<input type="hidden" name="units" value="imperial">`